modules/core/src/lib/mode-handlers/draw-polygon-handler.js

```diff
@@ -33,6 +33,12 @@ export class DrawPolygonHandler extends ModeHandler {
     let editAction: ?EditAction = null;
     const clickedEditHandle = getPickedEditHandle(picks);
 
+    if (clickedEditHandle) {
+      // remove the last element from the click sequence
```

Comment:
would be nice to add more info about why we have to remove it.

Reply:
Good idea